### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by azangru**
It's been a long several weeks since we last updated our dependencies.

After update, the tests pass and the app seems to work fine on dev machine.

### CODE REVIEW

File: src/ensembl/package.json [hunk 1]
```diff
@@ -68,76 +68,76 @@
...
     "html-webpack-plugin": "3.2.0",
-    "husky": "1.3.1",
+    "husky": "2.2.0",
     "identity-obj-proxy": "3.0.0",
     "image-webpack-loader": "4.6.0",
-    "jest": "24.7.1",
+    "jest": "24.8.0",
     "jest-fetch-mock": "2.1.2",
-    "jest-snapshot": "24.7.1",
+    "jest-snapshot": "24.8.0",
```
Comment (imran-ebi) [inline src/ensembl/package.json]: I thought we got rid of this package 🤔

Comment (azangru) [reply src/ensembl/package.json]: Turns out, we didn't 😄 
Should we?

Comment (imran-ebi) [reply src/ensembl/package.json]: Yes please

**Comment by imran-ebi** [comment]
The update was smooth for me, however the tests run a bit slower now at ~13s. Everything else works fine.

File: src/ensembl/package.json (hunk 1 above)
Comment (azangru) [reply src/ensembl/package.json]: Removed.

**Pull request merged**